ddtrace/internal/telemetry/data/metrics.py

```diff
@@ -0,0 +1,58 @@
+import time
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+from ...hostname import get_hostname
+
+
+class Series:
+    """
+    Stores and sends metrics using the Datadog Metrics API: https://docs.datadoghq.com/api/latest/metrics/
+    """
+
+    GAUGE = "gauge"
+    COUNT = "count"
+    RATE = "rate"
+
+    def __init__(self, metric, metric_type=COUNT, common=False, interval=None):
```

Comment:
hmm, I didn't realize before that `common` is actually part of the API spec... 🤔 

probably ok to leave this as-is, once we start adding a bunch of metrics we can re-think how we manage `common`.

Reply:
Yup I completely forgot that the dd.apm_telemetry.x.x.x prefix was added in the intake service if it is not set in the tracer. So all we would need to do is set a metric name and state whether it's common or not

Reply:
A description for these properties would be nice.